src/runnables/hyperparams_optimization.py

```diff
@@ -2,9 +2,14 @@
 import torch
 from torch_geometric.loader import DataLoader as pyg_DataLoader
 import tqdm
-from train_homo import train_batch, evaluate_batch
-from utils import get_gnn, load_data, get_criterion
-from utils_homo import normalize_data
+import os
+import sys
+# local imports
+# add parent directory to sys.path
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
I like to keep the imports in alphabetical order so it's easier to see if something is already imported